packages/ckeditor5-table/src/utils/table-properties.ts

```diff
@@ -125,3 +130,39 @@ export function getNormalizedDefaultProperties(
 
 	return normalizedConfig;
 }
+
+/**
+ * Returns the normalized default table properties.
+ */
+export function getNormalizedDefaultTableProperties(
+	config: Partial<NormalizedDefaultProperties> | undefined,
+	options?: NormalizeTableDefaultPropertiesOptions
+): NormalizedDefaultProperties {
+	return getNormalizedDefaultProperties( {
+		// It adds support for border none in the table element, keep it in sync with the content styles
+		// See more: https://github.com/ckeditor/ckeditor5/issues/6841#issuecomment-1959195608
+		borderStyle: 'double',
+		borderColor: 'hsl(0, 0%, 70%)',
+		borderWidth: '1px',
+		width: '100%',
+		height: '100%',
```

Comment:
I don't know if this is valid. By default, a table has no dimensions set.

Reply:
Yep looks like originally introduced [here](https://github.com/ckeditor/ckeditor5/issues/6841#issuecomment-1959195608). Works as expected without such styling. Thx